cmd/brew-find-formulae-to-bottle.rb

```diff
@@ -0,0 +1,40 @@
+module Homebrew
+  module_function
+
+  def head_is_merge_commit?
+    `git log --oneline --format=%s -1`.chomp == "Merge branch homebrew/master into linuxbrew/master"
+  end
+
+  def head_has_conflict_lines?
+    `git log --format=%b -1`.chomp.include?("Conflicts:") ||
+      `git log --format=%b -1`.chomp.include?("Formula/")
+  end
+
+  def strip_whitespace(line)
+    line.rstrip! && line.lstrip!
```

Comment:
I think `line.strip` is sufficient -- there is no need to do things "in place" (as we return things) and `strip` behaves correctly (returns a string if nothing was stripped out)

Reply:
Good shout. I had it in my head that `.strip` only did trailing whitespace for some reason.

```suggestion
    line.strip
```

Reply:
is there a chance you don't need this function at all? I think `line.strip!` will strip the string in-place correctly

Reply:
Yep, that’s what I’m reverting back to!

Reply:
OK